packages/lesswrong/components/editor/EditorFormComponent.jsx

```diff
@@ -215,26 +234,36 @@ class EditorFormComponent extends Component {
     this.setState({markdownValue: newContent})
     
     if (changed)
-      this.maybeSaveBackup();
+      this.afterChange();
   }
   
-  maybeSaveBackup = () => {
+  afterChange = () => {
+    this.hasUnsavedData = true;
+    
+    setTimeout(() => {
```

Comment:
`_.throttle` seems like a better fit here, since it uses an existing abstraction and allows us to remove some of the other variables we need to keep track of